canonical-web-and-design/ubuntu.com
# Vanilla 2 update

## Done

- Upgraded u.c to Vanilla 2.3.0 - these commits have been reviewed in previous PRs before being merged into this feature branch
- Also fixed a smaller issue on /blog/press-centre where the header was missing a suru

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
  - [ ] This affects the entire site, check as many pages as possible and ensure nothing looks off
  - [ ] visit /blog/press-centre, see that there is a suru behind the text, and that the text is readable

Fixes #5714
